src/bayes.h

```diff
@@ -1495,13 +1495,14 @@ typedef struct matrix
     int row;
     } Matrix;
 
+#define MAX_CHAR_STATES 36  // 0-9 a-z
```

Comment:
Move to bayes.h?

Reply:
it is already in bayes.h. you seems commenting on a outdated version